### PR TITLE
add aughighlights

### PR DESCRIPTION
**Pull request by jongio**

### CODE REVIEW

**Comment by jongio** [comment]
Don't merge yet.  Thanks

**Comment by jongio** [comment]
@joshfree @ramya-rao-a @schaabs @czubair @AlexGhiondea @lmazuel @mayurid - Please add comments to this PR for highlights to be included in Aug release notes and blog.  I primed it with the Azure Identity highlights.

File: releases/2020-08/index.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ tags: release
 sidebar: releases_sidebar
 ---
 
+## Highlights
+
+* Azure Identity GA release with new Developer Credential types, logging enhancements, proactive token refresh, and authority host support.
```
[the remark this answers is not in the document]
Comment (ramya-rao-a) [reply releases/2020-08/index.md:10]: The part about ObjectSerializer and JsonSerializer also applies to .Net right?

[the remark this answers is not in the document]
Comment (ramya-rao-a) [reply releases/2020-08/index.md:10]: The Key Vault one applies to all languages

[the remark this answers is not in the document]
Comment (ramya-rao-a) [reply releases/2020-08/index.md:10]: Form Recognizer is planning to go GA next week, so perhaps we can add that too?

[the remark this answers is not in the document]
Comment (jongio) [reply releases/2020-08/index.md:10]: What day?

[the remark this answers is not in the document]
Comment (jongio) [reply releases/2020-08/index.md:10]: Yes, if cross-language, like the Identity GA, then we don't need to call out "Java" in the highlight.

[the remark this answers is not in the document]
Comment (kaerm) [reply releases/2020-08/index.md:10]: Form Recognizer going GA cross language most probably Wednesday 8/19 (if all goes well)

[the remark this answers is not in the document]
Comment (heaths) [reply releases/2020-08/index.md:10]:
```suggestion
* Azure Identity GA release with new Developer Credential types, logging enhancements, proactive token refresh, and authority host support.
* Azure Cognitive Search SDK for .NET added `FieldBuilder` to help easily build a search index from a model type.
```

[the remark this answers is not in the document]
Comment (ramya-rao-a) [reply releases/2020-08/index.md:10]: @pakrym, @alzimmermsft, Can you provide some text for the ObjectSerializer/JsonSerializer parts?

**Comment by jongio** [comment]
@annatisch @schaabs @yiliuTo @kushagraThapar @sadasant @alzimmermsft @conniey @JoshLove-msft @heaths @tg-msft @mssfang @xiangyan99 @kinelski @samvaity @kristapratico @KieranBrantnerMagee @jsquire  @pakrym 

You all added Release Notes for August release.  Going forward we are also going to have a "Highlights" section on the Index page of the release.  Please either PR this PR or add a comment for anything that you would like to have highlighted. 

Thanks!
Jon

File: releases/2020-08/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jongio) [reply releases/2020-08/index.md:10]: For Form Recognizer, let's wait until we have a confirmed date.  Let's get this out there and then we'll adjust when they are ready.

**Comment by yiliuTo** [comment]
Hi @jongio , thanks for the reminder. And where will the highlights be displayed? Thanks

**Comment by yiliuTo** [comment]
Hi, @jongio may I still update highlights now by making another PR?

File: releases/2020-08/index.md [hunk 2]
```diff
@@ -5,6 +5,13 @@ tags: release
...
+* Azure Cognitive Search SDK for .NET added `FieldBuilder` to help easily build a search index from a model type.
+* Added ObjectSerializer and JsonSerializer APIs to support pluggable serialization within SDKs.
```
Comment (ramya-rao-a) [inline releases/2020-08/index.md:12]: @jongio It is worth calling out that the serializer work is done only in Java and .Net

Comment (jongio) [reply releases/2020-08/index.md:12]: It's fine. I think we let this one slide.

**Pull request merged**